lib/webpacker/dev_server.rb

```diff
@@ -0,0 +1,73 @@
+# Same convention as manifest/configuration.rb
+
+# Loads webpacker configuration from config/webpacker.yml
+
+require "webpacker/configuration"
+
+class Webpacker::DevServer < Webpacker::FileLoader
+  class << self
+    def enabled?
+      case ENV["WEBPACKER_DEV_SERVER"]
```

Comment:
Perhaps we reverse the behaviour with env variable - like `DISABLE_DEV_SERVER`? In case of webpacker this is enabled by default so not needed anyway.